modules/24_trade/se_trade/datainput.gms

```diff
@@ -180,16 +180,34 @@ $ifthen.import_h2_EU "%cm_Ger_Pol%" == "ensec"
   p24_seTradeCapacity(t,regi,regi2,entySe)$(t.val eq 2030) = p24_seTrade_Quantity(regi,regi2,entySe)*0.2;
 $endif.import_h2_EU
 
+*** parameters used in cm_import_EU scenarios nzero, nzero_bio and high_bio
+
+*** calculating the share of FE demand per carrier for each region in each region group
```

Comment:
I would reformulate to "calculate regional share of each region in the total of region group ext_regi, with respect to FE demand, for each aggregated carrier (liquids, gases, ...)"